packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -177,6 +177,15 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
 
   // Sync loading state into the layout.
   layout.isLoading = isLoading;
+
+  let isVerticalScrollbarVisible = false;
+  let isHorizontalScrollbarVisible = false; // do we need this one? can listviews horizontally scroll?
```

Comment:
IMO ListViews shouldn't support horizontal scrolling, seems kinda odd that you'd have a single column of content yet don't reserve enough room to render the full width of it. Happy to discuss though, maybe there is a use case for that out there

Reply:
Agreed, seems better to not have horizontal scrolling

Reply:
see
![Screen Shot 2022-04-28 at 2 57 58 PM](https://user-images.githubusercontent.com/698229/166997902-515e7868-7847-48f9-ba82-d827d9d44ba1.png)
which is story  https://reactspectrum.blob.core.windows.net/reactspectrum/c81848f00e7dde18beecc4cb16e38aa2a9fae89f/storybook/index.html?path=/story/listview--dynamic-items-small-viewport

Reply:
Right, I see that there are buttons that are out of view due to the ListView's width. However, should we support horizontal scrolling in that case or would setting an appropriate ListView width be the end user's responsibility? Perhaps it would be best to just support horizontal scrolling to make it easier on the end user

One possible issue that could arise is with text only cells and keyboard navigation, using left/right arrows wouldn't cause a scroll to happen.

Reply:
it's not too hard to get into a horizontally scrolling situation, if you have a few things in a list item and go to 400% zoom, it can occur pretty quickly.

wouldn't scrolling happen if the thing focus was moving to was off the screen?

Reply:
Ah good point about the zoom level, in that case we should definitely support horizontal scrolling, definitely something for a separate PR though.

As for the keyboard nav scrolling, if the row only contains text content the only thing focusable is the ListView row and thus left/right wouldn't move focus to something off screen and thus no scrolling will happen. Text truncation/wrapping might make this a non-issue but something keep in mind especially with long strings from localization